tests/pipeline/test_downsampling.py

```diff
@@ -750,14 +733,14 @@ def test_downsampled_rank(self):
         downsampled_rank = self.factor.rank().downsample('month_start')
         pipeline = Pipeline({'rank': downsampled_rank})
 
-        results_month_start = self.pipeline_engine.run_pipeline(
+        results_month_start = self.seeded_random_engine.run_pipeline(
```

Comment:
The seeded random loader is never actually used in this example (that's probably an argument against using that fixture, since that's not clear)